client/src/js/modules/types/mx/samples/sample-table-mixin.js

```diff
@@ -52,6 +52,16 @@ export default {
           SAFETYLEVEL: item.SAFETYLEVEL
         }));
     },
+    greenProteinsList() {
+      return this.$proteins()
+        .filter(item =>
```

Comment:
Is it worth it performing this check in the backend too? Just in case someone tries to "push through anyway"

(On a slightly less useful note, I think we can use `reduce` to perform this in O(N) (N operations, N being the array length), but that's just being pedantic and I don't think we gain much performance anyway)

Reply:
I wasn't sure if any other function was using the backend function, so didnt want to change it to stop Yellow/Red proteins. It also does each sample in turn on the backend, so you get a situation where some might succeed and then others fail.

Reply:
That makes sense! And it should be similar to changing it manually, so it's not a major risk to my eyes